tools/sim/lib/can.py

```diff
@@ -66,24 +63,3 @@ def can_function(pm, speed, angle, idx, cruise_button, is_engaged):
       msg.append([k, 0, b'\x00'*v, 0])
 
   pm.send('can', can_list_to_can_capnp(msg))
-
-def sendcan_function(sendcan):
```

Comment:
why was this deleted?

Reply:
That was only used in longitudinal tests?

Reply:
this is for carla